src/constants.ts

```diff
@@ -69,9 +69,11 @@ export const DEFAULT_HOST = '127.0.0.1';
 export const DEFAULT_PORT_P2P = 7667;
```

Comment:
Please double check the value for `Q96_ZERO` with the SDK/research team. Seems incorrect.